apps/dashboard/src/components/reports/TableScoreTag.vue

```diff
@@ -126,6 +133,24 @@ function handleToolTip(_taskId, _toolTip, _colData) {
   return _toolTip;
 }
 
+function handleSubskillToolTip(_taskId, _subskillId, _toolTip, _colData) {
+  const subskillInfo = _colData.scores?.[_taskId]?.[_subskillId];
+
+  if (_taskId === 'roam-alpaca') {
+    if (subskillInfo?.supportLevel) {
+      _toolTip += subskillInfo?.supportLevel + '\n' + '\n';
+      _toolTip += getFlags(_colData, _taskId);
+    }
+    _toolTip += 'Num Correct: ' + (subskillInfo?.rawScore || 0) + '\n';
+    _toolTip += 'Num Attempted: ' + (subskillInfo?.numAttempted || 0) + '\n';
```

Comment:
```suggestion
    _toolTip += 'Num Attempted: ' + (subskillInfo?.totalNumAttempted || 0) + '\n';
```
I think that the property is supposed to be `totalNumAttempted`, based on the run data in Firestore.
<img width="456" height="323" alt="image" src="https://github.com/user-attachments/assets/5c80d09e-10fe-43ac-a905-2239392d5764" />


Additionally, is it safe to set this to zero as a fallback value? If we attempt to calculate percent correct as numCorrect/numAttempted, that's going to produce a `NaN`, as seen below.



<img width="2376" height="666" alt="image" src="https://github.com/user-attachments/assets/c51404a8-5667-4d99-9014-a4b5f3983ee0" />

Reply:
Also regarding the fallback value, that fallback is for the tooltip only, just to show a 0 if that score does not exist. However, you are right about dividing by zero. The score report you were looking at has NaNs because the subscores changed variable names (totalNumCorrect -> numCorrect). So in your case the denominator was undefined (javascript returns `Infinity` for divide-by-zero ops). This does not need backwards compatibility as the previous subscore format was introduced less than a month ago and was not given to any real user. 
That being said, to be safe, we added a check for the existence and non-zero-ness of the denominator for the percentile score.